fdbmonitor/fdbmonitor.cpp

```diff
@@ -197,17 +197,46 @@ const char* get_value_multi(const CSimpleIni& ini, const char* key, ...) {
 	const char* ret = nullptr;
 	const char* section = nullptr;
 
+	std::string keyCopy(key);
```

Comment:
Can we rename this to something like `keyWithUnderscores` for clarity?

Reply:
Of course we can. Let me rename this one and the corresponding variable name in the FDBService